app/src/views/GrantRoundDetails.vue

```diff
@@ -184,6 +187,26 @@ function useGrantRoundDetail() {
     grantRound.value ? _grantRoundMetadata.value[grantRound.value.metaPtr] : null
   );
 
+  // --- Check properties ---
+  const hasWebsite = computed(() =>
+    grantRound.value
+      ? _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.projectWebsite !== undefined &&
+        _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.projectWebsite !== ''
+      : false
+  );
+  const hasGithub = computed(() =>
+    grantRound.value
+      ? _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.projectGithub !== undefined &&
+        _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.projectGithub !== ''
+      : false
+  );
+  const hasHandle = computed(() =>
+    grantRound.value
+      ? _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.twitterHandle !== undefined &&
+        _grantRoundMetadata.value[grantRound.value.metaPtr]?.properties?.twitterHandle !== ''
+      : false
+  );
```

Comment:
Since these definitions are duplicated again here, they should be moved to be computed properties of the `data.ts` store instead